content/docs/2.6/scalers/datadog.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "Datadog"
+layout = "scaler"
+availability = "v2.6+"
+maintainer = "Community"
+description = "Scale applications based on Datadog."
+go_file = "datadog_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `datadog` trigger that scales based on a Datadog metric.
+
+```yaml
+triggers:
+- type: datadog
+  metadata:
+    # Required: datadog metric query
+    query: "sum:trace.redis.command.hits{env:none,service:redis}.as_count()"
+    # Required: according to the number of query result, to scale the TargetRef
+    queryValue: "7"
+    # Optional: (Global or Average). Whether the target value is global or average per pod. Default: Average
+    type: "Global"
+    # Optional: The generated metric name would be datadog-redis-hits
+    metricName: "redis-hits"
+    # Optional: The time window (in seconds) to retrieve metrics from Datadog. Default: 90
+    age: "60"
+```
+
+**Parameter list:**
+
+- `query` - Required. The Datadog query to run.
+- `queryValue` - Required. Value to reach to start scaling.
+- `type`- Optional. Global or Average. Whether to start scaling based on the value or the average between pods. Default: Average.
+- `metricName` - Required. Name to identify the Metric in the external.metrics.k8s.io API. If using more than one trigger it is required that all `metricName`(s) be unique.
+- `age` - Optional. The time window (in seconds) to retrieve metrics from Datadog. Default: 90. 
+
+### Authentication
+
+Datadog requires both an API key and an APP key to retrieve metrics from your account.
+
+You should use `TriggerAuthentication` CRD to configure the authentication:
+
```

Comment:
I think we are missing the parameters here to provide? They are in the sample but not listed here. Please have a look at other scalers how to approach this.

Reply:
Thanks for catching this! Not sure how it happened, but I would had sworn I had added it. Added it now in a newer commit.